flottform/where-are-you-at/src/routes/feedback/+page.svelte

```diff
@@ -0,0 +1,324 @@
+		<form class="grid grid-cols-1 gap-4 max-w-screen-sm" bind:this={form} method="POST">
+			<p class="italic text-xs">
+				* Please know that we use your data solely for internal analysis and improvement of our
+				service, and it's stored securely in a private GitHub project. We won't share your
+				information with other third parties or use it for marketing purposes. There will be no
+				promotional mailings or advertising campaigns targeted based on your feedback. We'll only
+				contact you if you've provided us with your contact details.
+			</p>
+			<div class="flex flex-col gap-2">
+				<label for="name">Name <span aria-hidden="true" class="text-sm text-red-700">*</span></label
+				>
+				<input
+					type="text"
+					name="name"
+					id="name"
+					required
+					class="border border-primary-blue rounded px-2 py-1"
+				/>
+			</div>
+			<div class="flex flex-col gap-2">
+				<fieldset>
+					<legend>What's your preferred way of contact?</legend>
+					<div class="grid grid-cols-3 gap-1">
+						<label
+							for="contactChoice1"
+							class="relative z-10 px-4 py-4 rounded text-center h-full w-full group hover:cursor-pointer"
+						>
+							<input
+								type="radio"
+								id="contactChoice1"
+								name="contact"
+								value="email"
+								bind:group={contactChoice}
+							/>
+							E-Mail
+						</label>
+						<label
+							for="contactChoice2"
+							class="relative z-10 px-4 py-4 rounded text-center h-full w-full group hover:cursor-pointer"
+						>
+							<input
+								type="radio"
+								id="contactChoice2"
+								name="contact"
+								value="linkedin"
+								bind:group={contactChoice}
+							/>
+							LinkedIn
+						</label>
+						<label
+							for="contactChoice3"
+							class="relative z-10 px-4 py-4 rounded text-center h-full w-full group hover:cursor-pointer"
+						>
+							<input
+								type="radio"
+								id="contactChoice3"
+								name="contact"
+								value="twitter"
+								bind:group={contactChoice}
+							/>
+							X / Twitter
+						</label>
+					</div>
+				</fieldset>
+			</div>
+
+			<div class="flex flex-col gap-2">
+				<label for="email" class="flex flex-col gap-2">E-Mail</label>
+				<input
+					type="email"
+					name="email"
+					id="email"
+					class="border border-primary-blue rounded px-2 py-1"
+				/>
+			</div>
+
+			<div class="flex flex-col gap-2">
+				<label for="linkedin" class="flex flex-col gap-2">LinkedIn</label>
+				<input
+					type="text"
+					name="linkedin"
+					id="linkedin"
+					class="border border-primary-blue rounded px-2 py-1"
+				/>
+			</div>
+
+			<div class="flex flex-col gap-2">
+				<label for="twitter" class="flex flex-col gap-2">X / Twitter</label>
+				<input
+					type="text"
+					name="twitter"
+					id="twitter"
+					class="border border-primary-blue rounded px-2 py-1"
+				/>
+			</div>
+
+			<div class="flex flex-col gap-2">
+				<label for="feedbackPositive"
+					>What do you like about Flottform?
```

Comment:
We should change 'Flottform' to 'where-are-you.at' or 'Location Tracker'

```suggestion
					>What do you like about where-are-you.at?
```